src/containers/Noticias.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { firebaseConnect } from 'react-redux-firebase';
+
+import NoticiasList from '../components/NoticiasList';
+
+const Noticias = ({ firebase, noticias }) => (
+  <section>
+
+    <div className='grid-row'>
+      <div className='grid-item item-s-3'>
+        <h2>Noticias</h2>
```

Comment:
These kind of list view layouts are going to be quite common. Can we make this flexible for all post types?

Reply:
Sure, once we get 2-3 other pages would be good to find an abstraction.